fairlearn/widget/FairlearnWidget.py

```diff
@@ -0,0 +1,23 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Defines the python side of the shared state of the Fairlearn widget."""
```

Comment:
fairlearn all lower case please here and in every file, excluding class names, of course

Reply:
will do. Question though, the design has "FairLearn" in the title displayed of the widget in several places. Is the plan fairlearn in all code, but its ok in presentation?